SUMMARY.md

```diff
@@ -2,81 +2,107 @@
+* [How Apostrophe modules are structured](tutorials/advanced-development/how-apostrophe-modules-are-structured.md)
+* [Promise Events](tutorials/advanced-development/03-promise-events/promise-events.md)
+* [Command line tasks](tutorials/advanced-development/04-apostrophe-cli/command-line-tasks.md)
+* [Internationalization and localization](tutorials/advanced-development/05-internationalization-localization/apostrophe-i18n-config.md)
+* [The Database](tutorials/advanced-development/06-database/README.md)
+  * [Apostrophe's model layer: working with the database](tutorials/advanced-development/06-database/01-model-layer.md)
+  * [Working with cursors](tutorials/advanced-development/06-database/02-cursors.md)
+  * [Accessing the database directly](tutorials/advanced-development/06-database/03-accessing-the-database-directly.md)
+  * [Using MongoDB replica sets with Apostrophe](tutorials/advanced-development/04-database/replica-set.md)
+* [Guide to schemas](tutorials/advanced-development/07-schemas/schema-guide/README.md)
+  * [Custom schema fields for pages](tutorials/schema-guide/schema-guide.md)
+  * [Custom schema field types](tutorials/schema-guide/custom-schema-field-types.md)
 
-## Technical Overviews
+## HOWTOS
+* [How do I create a '404 not found' page?](tutorials/howtos/notFoundPage.md)
+* [Configuring CKEditor in Apostrophe](tutorials/howtos/ckeditor.md)
+* [Building combined menus and custom buttons with the admin bar](tutorials/howtos/admin-bar.md)
 
-* [Technical overview: 30,000 feet](technical-overviews/technical-overview.md)
-* [How Apostrophe starts up](technical-overviews/how-apostrophe-starts-up.md)
-* [How Apostrophe handles requests](technical-overviews/how-apostrophe-handles-requests.md)
-* [How Apostrophe modules are structured](technical-overviews/how-apostrophe-modules-are-structured.md)
+## Apostrophe DevOps: Deployment
+* [Hosting Apostrophe in production](tutorials/intermediate/deployment.md)  
+* [Deploying Apostrophe to a Linode linux server quickly](tutorials/howtos/linode.md)
+* [Deploying Apostrophe in the Cloud with Heroku](tutorials/howtos/deploying-apostrophe-in-the-cloud-with-heroku.md)
+* [Deploying Apostrophe with AWS and Elastic Beanstalk](tutorials/howtos/deploying-apostrophe-in-the-cloud-with-aws.md)
+* [Running Apostrophe on multiple cores and/or servers](tutorials/howtos/multicore.md)
+* [Running Apostrophe on Windows](tutorials/howtos/windows.md)
+* [Building Docker images for Apostrophe projects](tutorials/howtos/docker.md)
+* [Storing images and files in Amazon S3](tutorials/howtos/storing-images-and-files-in-amazon-s3.md)
+* [Storing images and files in Azure Blob Storage](tutorials/howtos/storing-images-and-files-in-azure-blob-storage.md)
+* [Storing images and files in Google Cloud Storage](tutorials/howtos/storing-images-and-files-in-google-cloud-storage.md)
+
+
+## Apostrophe Configuration and Administration
```

Comment:
I feel like this heading is the least clear right now. What do email, open graph, user notifications, redis sessions, single sign-on, and redirects after login have in common? There's no administration here, and everything is configuration. I don't know the answer either.

Reply:
This one isn't done yet which is why it's still a bit mixed up.

Reply:
So for now I think the additional Configuration type items are going back into HOWTOs. I think as the docs grow, there will be a need for a separate section, but I think having a small HOWTO section with a little bit of a hodgepodge of info will work for now.